File: app/src/main/java/com/mithilakshar/mithilapanchang/calendarviewholder.java
package com.mithilakshar.mithilapanchang;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;


public class calendarviewholder extends RecyclerView.ViewHolder {

    TextView calendardayText,calendardateText,calendardescText;

    public calendarviewholder(@NonNull View itemView) {

        super(itemView);

        calendardayText=itemView.findViewById(R.id.calendardayText);
        calendardateText=itemView.findViewById(R.id.calendardateText);
        calendardescText=itemView.findViewById(R.id.calendardescText);


    }
}

File: app/src/main/java/com/mithilakshar/mithilapanchang/calendarviewholder.kt
package com.mithilakshar.mithilapanchang

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView

class calendarviewholder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var calendardayText: TextView
    var calendardateText: TextView
    var calendardescText: TextView

    init {
        calendardayText = itemView.findViewById(R.id.calendardayText)
        calendardateText = itemView.findViewById(R.id.calendardateText)
        calendardescText = itemView.findViewById(R.id.calendardescText)
    }
}
